// @progress/kendo-datasource-vue-wrapper/dist/npm/KendoPivotDataSource/KendoPivotDataSourceMixin.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _KendoDataSourceBase = require('../KendoDataSourceBase');

var _KendoDataSourceBase2 = _interopRequireDefault(_KendoDataSourceBase);

var _PivotDataSourceProps = require('./PivotDataSourceProps');

var _PivotDataSourceProps2 = _interopRequireDefault(_PivotDataSourceProps);

var _KendoPivotColumn = require('../KendoPivotColumn');

var _KendoPivotColumn2 = _interopRequireDefault(_KendoPivotColumn);

var _KendoPivotMeasure = require('../KendoPivotMeasure');

var _KendoPivotMeasure2 = _interopRequireDefault(_KendoPivotMeasure);

var _KendoPivotRow = require('../KendoPivotRow');

var _KendoPivotRow2 = _interopRequireDefault(_KendoPivotRow);

var _Schema = require('./Schema');

var _Schema2 = _interopRequireDefault(_Schema);

var _Transport = require('./Transport');

var _Transport2 = _interopRequireDefault(_Transport);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
    props: _PivotDataSourceProps2.default,
    render(h) {
        return h('div', this.$slots.default);
    },
    mixins: [_KendoDataSourceBase2.default, _Schema2.default, _Transport2.default],
    mounted() {
        this._resolveChildren();

        this.kendoDataSource = new kendo.data.PivotDataSource(this.widgetOptions);
    },
    methods: {
        kendoWidget() {
            return this.kendoDataSource || new kendo.data.PivotDataSource(this.widgetOptions);
        },
        _resolveChildren() {
            this.resolveChildren('columns', _KendoPivotColumn2.default.name);
            this.resolveChildren('measures', _KendoPivotMeasure2.default.name);
            this.resolveChildren('rows', _KendoPivotRow2.default.name);
        }
    }
};
